book/src/specs/urs.md

```diff
@@ -68,7 +68,7 @@ TODO: specify the encoding
 
 ### Vesta
 
-**`Gs`**. 
+**`Gs`**.
```

Comment:
We had so many trailing spaces, I wonder why it's not enforced through CI/github handles...